src/extraction.js

```diff
@@ -111,12 +113,46 @@ module.exports = {
       extractJob.on('error', reject);
     });
 
+    // destinationUriFileCounts is an array where each value corresponds to
+    // a URL provided in destinationUri in the job. We only ever give one URL,
+    // so we only get one value.
+    const outputFileCount =
+      extractJob.metadata.statistics.extract.destinationUriFileCounts[0];
+
     console.log(
       JSON.stringify({
         ExtractJob: { Type: extractionType, Metadata: extractJob.metadata },
       }),
     );
 
-    await MakeCopies(extractionType, config, bucketName, objectName);
+    const copyMachines = [];
+
+    // Because the extaction could have generated multiple files, we need to
+    // copy however many were created.
+    for (let i = 0; i < outputFileCount; i++) {
+      // BigQuery makes files with 12-digit numbers, starting at 000000000000
```

Comment:
This is fine for now, but feels slightly brittle to be relying on?

Reply:
The choices seemed to be either: trust that BQ reports the correct number and create the file names based on that, or ignore what BQ says and do a `list` of some GCS prefix and pattern match to find the files that look like they belong to this job. Or maybe something else I'm not thinking about.

Reply:
If the docs don't specify one way or the other, probably safest to list the entire directory.  But then again, this gets us off the ground, and it seems unlikely to break.